src/community/oseo/oseo-rest/src/main/java/org/geoserver/opensearch/rest/OgcLink.java

```diff
@@ -20,16 +25,25 @@
 
     String href;
 
+    Map<String, String> unknownFields = new HashMap<>();
```

Comment:
I believe it should be ``Map<String, Object>``, see comments above about respecting types.

Reply:
done